src/modules/ShortcutGuide/ShortcutGuide.Ui/ShortcutGuideXAML/OOBEView.xaml

```diff
@@ -0,0 +1,47 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Page
+    x:Class="ShortcutGuide.ShortcutGuideXAML.OOBEView"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:ShortcutGuide.ShortcutGuideXAML"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <StackPanel Margin="32,24" Spacing="12">
+        <TextBlock x:Uid="OOBETitle" Style="{StaticResource TitleTextBlockStyle}" />
+        <TextBlock
+            x:Uid="OOBEDescription1"
+            Foreground="{ThemeResource TextFillColorSecondaryBrush}"
+            TextWrapping="WrapWholeWords" />
+        <StackPanel Orientation="Horizontal">
```

Comment:
It may be much simpler to put all the text in one string. Also helps if somebody wants to change things later.

Reply:
Fair point. Just thinking out loud.